src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+
+                    {milestoneBadge}
+                </div>
+            </div>
+        </li>;
+    }
+}
+
+class IssueList extends React.Component<{ id: string, issues: Data.Issue[], emptyText: string, title: string, collapsable?: boolean }, undefined> {
+    render() {
+        let content;
+        let collapser;
+        if (this.props.issues.length > 0) {
+            content = <ul className="list-group">
+                {this.props.issues.map(issue => <IssueView key={issue.id} issue={issue} />)}
+            </ul>;
+            if (this.props.collapsable) {
+                collapser = <button className="btn btn-xs btn-info chevronButton" type="button" data-toggle="collapse" data-target={`#collapse-otherissues-${this.props.id}`}>
+                    <span id="chevronGlyph" className="glyphicon glyphicon-chevron-right"></span>
+                    {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                    Show/hide
+                </button>;
+                content = <div className="collapse" id={`collapse-otherissues-${this.props.id}`} style={{ "marginTop": "10px" }}>
+                    {content}
+                </div>;
+            }
+        }
+        else {
+            content = <div className="alert alert-info">{this.props.emptyText}</div>;
+        }
+
+        return <div>
+            <h4>
+                {this.props.title}
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                <span className="badge">{this.props.issues.length}</span>
+                {' ' /* https://facebook.github.io/react/blog/2014/02/20/react-v0.9.html#jsx-whitespace */}
+                {collapser}
+            </h4>
+            {content}
+        </div>;
+    }
+}
+
+type RateLimitUpdater = (graphQl: { cost: number, remaining: number, resetAt: string }, rest: { remaining: number, reset: string }) => void;
+
+interface PersonProps {
+    baseUrl: string,
+    repoSet: string,
+    login: string,
+    environment: string,
+    updateRateLimit: RateLimitUpdater;
+}
+
+class Person extends React.Component<PersonProps, { loading: boolean, error: string, data: Data.RepoSetIssueResult }> {
+    constructor(props: PersonProps) {
+        super(props);
+        this.state = {
+            loading: true,
+            error: "",
+            data: null
+        };
+    }
+
+    componentDidMount() {
+        // Can't await this here, but that's OK, it will call setState eventually.
+        this.reload();
+    }
+
+    async reload() {
+        // This doesn't immediately clear the loading widget, but it will when the function returns and we re-render.
+        this.setState({
+            loading: true,
+            error: "",
+            data: null
+        });
+
+        let resp = await fetch(`${this.props.baseUrl}api/repoSets/${this.props.repoSet}/issues/${this.props.login}`, fetchSettings)
```

Comment:
Should also have a `try`/`catch` around this, because the `fetch` promise can also be rejected if there's a network error (as opposed to if you get an HTTP response with some status like 500, where the promise succeeds but `resp.ok` is false).